runtime/syntax/raku.yaml

```
@@ -1,7 +1,7 @@
 filetype: raku
 
 detect:
-    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku$|\\.rakumod$|\\.rakudoc$)"
+    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku$|\\.rakumod$|\\.rakudoc$|\\.rakutest$)"
```

```suggestion
    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku(mod|doc|test)?$|\\.nqp$)"
```

It shortens the line and adds `.nqp` for completeness.